### DISCUSSION TITLE
캐시의 지역성에 대해 설명해주세요.

### DISCUSSION THREAD

**Discussion by eatingbug** [Team-Psyduck/tech-sharing#64] [General]
- 메모리 계층 구조와 캐시 지역성 간의 관계는 어떻게 되나요? 캐시 지역성은 어떻게 이러한 계층 구조를 최적화하는데 도움을 주나요?

**Comment by eatingbug** [comment]
캐시가 효율적으로 동작하려면, 캐시에 저장할 데이터가 지역성을 가져야한다. 지역성이란 데이터 접근이 시간적 또는 공간적으로 가깝게 일어나는 것을 의미한다.

- 시간적 지역성
  - 특정 데이터가 한번 접근됐을 때, 가까운 미래에 또 한번 접근될 가능성이 높은 것을 시간적 지역성이라고 한다.
- 공간적 지역성
  - 특정 데이터와 가까운 주소가 순서대로 접근되었을 경우를 공간적 지역성이라고 한다.

출처 : https://ko.m.wikipedia.org/wiki/%EC%BA%90%EC%8B%9C

**Reply by kkg5** [reply]
캐시의 지역성과 관련해서 `퀵 정렬`이 `합병 정렬`, `힙 정렬`에 비해 빠른 이유가 무엇일까요?
